code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -7,6 +7,7 @@ h2fire = -3
 tritfire = -2
 halon_o2removal = -1
 nitrous_decomp = 0
+miasma_decomp = 0
```

Comment:
If you are doing this, don't use same numbers for two reactions, either put this at the end or move every number by one